Tapawingo_backend/Controllers/EventsController.cs

```diff
@@ -0,0 +1,27 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using Tapawingo_backend.Models;
+using Tapawingo_backend.Services;
+using System.Linq;
+
+namespace Tapawingo_backend.Controllers
+{
+    [Route("api/organisations/{organisationId}/[controller]")]
```

Comment:
Dit is niet hoe we dit doen. 
Je wil dit api/[controller] maken. 

Ik zou deze route weglaten, en [HttpGet("api/organisations/{organisationId}/Events")]